.changeset/tasty-beers-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
Let's summerize the improvements to a single changeset file and keep a single file.